packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -184,6 +184,10 @@ export class Messenger {
                     value: JDKVersion.JDK11,
                     label: JDKVersion.JDK11.toString(),
                 },
+                {
+                    value: JDKVersion.JDK17,
+                    label: JDKVersion.JDK17.toString(),
```

Comment:
For this and the 2 above, don't need to use `toString()`

Reply:
Oh ok I was just copying what was done above, are you just saying do "11" or "17?

Reply:
`JDKVersion.JDK17` is fine, we just don't need the `toString()` call here, and may as well fix it in the 2 places above too